src/main/java/com/iwaconsolti/league/Config/PlayerConfig.java

```diff
@@ -0,0 +1,21 @@
+package com.iwaconsolti.league.Config;
+import com.iwaconsolti.league.model.PlayerModel;
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.List;
+
+@Configuration
```

Comment:
Using Configurations (MatchConfig, TeamConfig, PlayerConfig) If Configs (like MatchConfig, TeamConfig, etc.) are not annotated with @Configuration, and are not declared as Spring beans, they could cause autoconfiguration issues.